2-core-rules.md

```diff
@@ -214,108 +209,97 @@ _Fleeing_ models must take a _leadership test_. If the test is passed, the model
+
+A model that runs, is _incapacitated_, or charges cannot hide that turn. Their sudden burst of speed does not give them time to hide. They also may not enter the _Hidden_ mode if there are enemies within 4", or if they are within unobstructed line of sight to an enemy.
 
-A model may stay hidden over several turns, so long as he stays behind a wall or similar feature. He may even move around so long as he stays hidden while doing so. If an enemy moves so that he can see the hidden warrior, the model is no longer hidden and the counter is removed. When hidden, a warrior cannot be seen, shot at or charged.
+A model may stay hidden over several turns, so long as they stay behind a wall or similar feature. They may even move around so long as they stay hidden while doing so. 
 
-While hiding, a model cannot shoot or cast spells without giving away its position. If a hidden model shoots, or moves so that he can be seen, he is no longer hidden and can be shot at as normal.
+While hiding, a model cannot shoot or cast spells without giving away its position. If a hidden model shoots, or moves so that the can be seen, he is no longer hidden and can be shot at as normal.
 
-A model may not hide if he is too close to an enemy model – he will be seen or heard no matter how well concealed. Enemy warriors will always see, hear or otherwise detect hidden foes within their Initiative value in inches. So a warrior whose Initiative value is 3 will automatically spot all hidden enemies within 3".
+If you wish to either shoot at, cast spells at, or charge a _Hidden_ model your warriors may attempt to detect them. A model's detection range is twice their Initiative value in inches: An inattentive Orc with an Initiative value of 2 would therefore only attempt to detect enemies within 4". An eagle-eyed Elf with an Initiative of 5 however may attempt to detect enemies within 10"! Detecting an enemy requires an Initiative test, and each warrior may only attempt to detect once per turn. An model that is detected immediately loses its _Hidden_ status.
 
 #### Terrain
 
 The ruined city of Mordheim is a dark and dangerous place, where tumbled down towers and blasted houses form a vast maze of streets and alleyways.
 
-OPEN GROUND: The tabletop surface, floors of buildings, connecting overhangs, ladders and ropes are all considered to be open ground and will not affect movement even if the model is charging. It can also go through doors and hatches without slowing down.
-
-DIFFICULT GROUND: Difficult ground includes steep or treacherous slopes, bushes and the angled roofs of buildings. Models move at half speed over difficult terrain.
+OPEN GROUND: The tabletop surface, floors of buildings, connecting overhangs, and horizontal gaps of less than 3". are all considered to be open ground and will not affect movement even if the model is charging. It can also go through doors and hatches without slowing down.
```

Comment:
```suggestion
__Open Ground:__ The tabletop surface, floors of buildings, connecting overhangs, and horizontal gaps of less than 3". are all considered to be open ground and will not affect movement even if the model is charging. It can also go through doors and hatches without slowing down.
```

Reply:
👍